research.md

```diff
@@ -39,6 +39,9 @@ Lots more hacks built using our API can be found at the [Perspective Hacks Galle
  
 ## Research Contributions
 
+* [Toxicity Detection: Does Context Really Matter?
+](https://www.aclweb.org/anthology/2020.acl-main.396/) studies the importance of  context, and when as well as to what extent it can change the perceived toxicity of posts, finding that context can both amplify or mitigate the perceived toxicity of posts, and moreover for a subset of the data that including context can even reverse the toxicity label.
+* [Classifying Constructive Comments](https://arxiv.org/abs/2004.05476) introduces a new dataset of constructive comments, defines a taxonomy of characteristics of constructiveness, and provides models for constructiveness trained on this dataset.
```

Comment:
This is a preprint - but I don't know if we can say much about where it's going to be published, maybe just (preprint)

Reply:
Appended "(preprint)" to it for now.